Zend/zend_vm_def.h

```diff
@@ -9808,6 +9808,44 @@ ZEND_VM_HANDLER(209, ZEND_INIT_PARENT_PROPERTY_HOOK_CALL, CONST, UNUSED|NUM, NUM
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(210, ZEND_FETCH_DEFAULT_ARG, UNUSED, UNUSED)
+{
+	USE_OPLINE
+	SAVE_OPLINE();
+
+	zend_function *called_func = EX(call)->func;
+
+	zend_string *reflection_class_name = ZSTR_INIT_LITERAL("ReflectionParameter", 0);
+	zend_class_entry *reflection_class = zend_fetch_class(reflection_class_name, ZEND_FETCH_CLASS_DEFAULT);
+	zend_string_release(reflection_class_name);
+
+	/*
+	 * [0]: The function (string) or class method (array) to reflect parameters from.
+	 * [1]: A zero-based integer specifying the parameter position.
+	 */
+	zval constructor_params[2];
+	ZVAL_LONG(&constructor_params[1], opline->op1.num - 1);
+	if (called_func->common.scope == NULL) {
+		ZVAL_STR_COPY(&constructor_params[0], called_func->common.function_name);
+	} else {
+		zval arr;
+		array_init_size(&arr, 2);
+		add_next_index_string(&arr, called_func->common.scope->name->val);
+		add_next_index_string(&arr, called_func->common.function_name->val);
+		ZVAL_ARR(&constructor_params[0], Z_ARR(arr));
+	}
+
+	zval reflection_obj;
+	// TODO: Check result.
+	/*zend_result res =*/ object_init_with_constructor(&reflection_obj, reflection_class, 2, constructor_params, NULL);
```

Comment:
You shouldn't need create a reflection instance to use its api. Just expose the relevant code called within getDefaultValue().

Reply:
What does *just expose* mean, exactly?

Reply:
Call the same functions that getDefaultValue() calls.